lib/flutter_secure_storage.dart

```diff
@@ -140,21 +139,21 @@ enum IOSAccessibility {
 
 class IOSOptions extends Options {
   IOSOptions(
-      {String groupId,
-      IOSAccessibility accessibility = IOSAccessibility.unlocked})
+      {String? groupId,
+      IOSAccessibility? accessibility = IOSAccessibility.unlocked})
```

Comment:
@SteveAlexander shouldn't `accessibility` be non-nullable?

Reply:
yes!